src/oscar/core/loading.py

```diff
@@ -58,6 +59,17 @@ def get_class(module_label, classname, module_prefix='oscar.apps'):
 
 
 def get_classes(module_label, classnames, module_prefix='oscar.apps'):
+    # Allow projects to supply their own logic for loading classes.
+    # If none is provided then use Oscar's default_class_loader
+    _custom_class_loader = getattr(settings, 'OSCAR_DYNAMIC_CLASS_LOADER', None)
+    if _custom_class_loader:
+        class_loader = import_string(_custom_class_loader)
```

Comment:
I feel like importing this afresh every time this function is called is bad form - but not sure how/where to cache it (bearing in mind that `functools.lru_cache` doesn't exist in Python 2). Any suggestions here would be appreciated.